### ISSUE TITLE
prefer-const fails on array destructurisation

### ISSUE THREAD

**Issue by cakeinpanic** [palantir/tslint#2903]
### Bug Report

- __TSLint version__: 4.5.1
- __TypeScript version__: 2.0.10
- __Running TSLint via__: (pick one) CLI 

#### TypeScript code being linted

```ts
let [a,b] = [1,2];
a++;
```

with `tslint.json` configuration:

```json
"prefer-const": [true, {"destructuring": "all"}],
```

#### Actual behavior
 lint fails cause b is never changed
#### Expected behavior
lint should not fail

**Comment by adidahiya** [comment]
That option was only added in v5+ https://github.com/palantir/tslint/commit/6b913e332f2acad11a1f1ac7d2a88ff4f4117348
